### ISSUE TITLE
Synax Error: expecting identifier 'end', not 'EOF' 

### ISSUE THREAD

**Issue by nbalsamo** [maiha/kafka.cr#2]
I have a very simple crystal script that simply requires the kafka library.

```
require "kafka"
```

When i attempt to run this I get the following error:
```
Error in src/test.cr:1: while requiring "kafka"

require "kafka"
^

in lib/kafka/src/kafka.cr:11: while requiring "./kafka/*"

require "./kafka/*"
^

in lib/kafka/src/kafka/commands.cr:1: while requiring "./protocol"

require "./protocol"
^

in lib/kafka/src/kafka/protocol.cr:1: while requiring "./protocol/*"

require "./protocol/*"
^

Syntax error in lib/kafka/src/kafka/protocol/macros.cr:115: expecting identifier 'end', not 'EOF'
```

**Comment by maiha** [comment]
Could you tell me your crystal version?
```shell
crystal -v
```

**Comment by nbalsamo** [comment]
```
Crystal 0.24.1 (2018-01-27)

LLVM: 5.0.1
Default target: x86_64-apple-macosx

```

**Comment by maiha** [comment]
@nbalsamo 
Sorry, I forgot to publish a tag. Just released `0.6.5`. Could you try it ?
